system-tests/__snapshots__/vite_dev_server_fresh_spec.ts.js

```diff
@@ -82,7 +82,7 @@ We dynamically generated a new test to display this failure.
 
   (Screenshots)
 
-  -  /XXX/XXX/XXX/cypress/screenshots/AppCompilationError.cy.jsx/An uncaught error wa     (1280x633)
+  -  /XXX/XXX/XXX/cypress/screenshots/AppCompilationError.cy.jsx/An uncaught error wa     (1280x581)
```

Comment:
Are these tests actually testing screenshot behavior? Can we just turn the screenshots off here?

(Not suggesting this optimization be a part of this PR, I could maybe follow up with a PR for this if so)

Reply:
K, I'll take a look at this.